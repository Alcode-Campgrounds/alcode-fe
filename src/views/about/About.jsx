import React from 'react'
import { Loader } from '../../components/loader/Loader'
import { useEffect, useState } from 'react'


export default function About() {
    const [loading, setLoading] = useState(true)
    useEffect(() => {setLoading(false)},[])
    console.log('loading', loading)
    return (
       <> 
       { loading ? <Loader /> : 
         (<div className='loading-container'>
            <h1>All about Alan, yo</h1>
        </div>)}
        </>
    )
}

     
