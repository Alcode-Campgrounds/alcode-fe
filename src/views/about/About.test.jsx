import React from 'react';
import { render } from '@testing-library/react';
import About from './About'


it('should render the about component', () => {
  const { container } = render( <About /> );

  expect(container).toMatchSnapshot();
});
