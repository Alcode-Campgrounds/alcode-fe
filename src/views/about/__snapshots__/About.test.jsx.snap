// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the about component 1`] = `
<div>
  <div>
    <h1>
      All about Alan, yo
    </h1>
  </div>
</div>
`;
